src/main/resources/swagger/api.yml

```diff
@@ -302,6 +302,7 @@ components:
       type: object
       required:
         - name
+        - token
```

Comment:
the "get project by token" endpoint should return some error if the project was already archived.